samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -0,0 +1,221 @@
+\n\t - CloudFormation Execution Role(IAM Role) that is assumed by CloudFormation to deploy the SAM application
+\n\t - Artifacts bucket(S3 bucket) to store the sam build artifacts
+\n\t - ECR repo for the container images of Lambda functions having PackageType property set to Image
+"""
+
+PIPELINE_CONFIG_DIR = os.path.join(".aws-sam", "pipeline")
+PIPELINE_CONFIG_FILENAME = "pipelineconfig.toml"
+
+
+@click.command("bootstrap", short_help=SHORT_HELP, help=HELP_TEXT, context_settings=dict(max_content_width=120))
+@configuration_option(provider=TomlProvider(section="parameters"))
+@click.option(
+    "--interactive/--no-interactive",
+    is_flag=True,
+    default=True,
+    help="Disable interactive prompting for bootstrap parameters, and fail if any required arguments are missing.",
+)
+@click.option(
+    "--stage-name",
+    help="The name of the corresponding pipeline stage. It is used as a suffix for the created resources.",
+    required=False,
+)
+@click.option(
+    "--pipeline-user",
+    help="The ARN of the IAM user having its AccessKeyId and SecretAccessKey shared with the CI/CD provider."
+    "It is used to grant this IAM user the permissions to access the corresponding AWS account."
+    "If not provided, the command will create one along with AccessKeyId and SecretAccessKey credentials.",
+    required=False,
+)
+@click.option(
+    "--pipeline-execution-role",
+    help="The ARN of an IAM Role to be assumed by the pipeline-user to operate on this stage. "
+    "Provide it only if you want to user your own role, otherwise, the command will create one",
+    required=False,
+)
+@click.option(
+    "--cloudformation-execution-role",
+    help="The ARN of an IAM Role to be assumed by the CloudFormation service while deploying the application's stack "
+    "Provide it only if you want to user your own role, otherwise, the command will create one",
+    required=False,
+)
+@click.option(
+    "--artifacts-bucket",
+    help="The ARN of a S3 bucket to hold the sam build artifacts. "
+    "Provide it only if you want to user your own S3 bucket, otherwise, the command will create one",
+    required=False,
+)
+@click.option(
+    "--create-ecr-repo/--no-create-ecr-repo",
+    is_flag=True,
+    default=False,
+    help="If set to true and no ecr-repo is provided this command will create an ECR repo to hold the image container "
+    "of the lambda functions having Image package type.",
+)
+@click.option(
+    "--ecr-repo",
+    help="The ARN of an ECR repo to hold the image containers of the lambda functions of image package type. "
+    "If Provided, the create-ecr-repo argument is ignored. If not provided and create-ecr-repo is set to true "
+    "The command will create one.",
+    required=False,
+)
+@click.option(
+    "--pipeline-ip-range",
+    help="If provided, all requests coming from outside of the given range(s) are denied.",
```

Comment:
It is not apparent what format customers should use here. Maybe provide some examples